Fix seeds.exs to meet not null contraints

fahrradflucht opened the pull request:
5e6b235c3078f9dbacfa5295e27b5b3fc770a1d0 introduced a not null constraint for the organization description.

This adds a description to the seeded org to meet that constraint.
## 

While investigating this I noticed that the organization model doesn't require a description on the changeset. Is that intentional?

JoshSmith commented:
I think it will be a `ProjectUpdate` record. Would love to get it into 1.0 but I think it needs more thought as of right now. Since it's just Code Corps on here for now, we can simply use the mailing list as a means of doing this.

JoshSmith commented:
👍

fahrradflucht committed:
Fix seeds.exs to meet not null contraints

5e6b235c3078f9dbacfa5295e27b5b3fc770a1d0 introduced a not null constraint for the organization description.

This adds a description to the seeded org to meet that constraint.

joshsmith commented:
> the organization model doesn't require a description on the changeset

@Fahrradflucht no, that's not intentional. Feel free to tackle that. Thanks for noticing!

joshsmith reviewed (approved):
👍

joshsmith commented:
Good catch. We also really need to add in some `OrganizationMembership` relationships and similar relationships, because those are currently missing. Leads to a lot of heartache.

Feel like there should be some easier ways to seed than what we're doing right now generally. All feels a bit hacky.
